eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java

```diff
@@ -217,6 +220,7 @@ private ImmutableBagMultimapFactory()
         public final MutableSetMultimapFactory set = new MutableSetMultimapFactory();
         public final MutableSortedSetMultimapFactory sortedSet = new MutableSortedSetMultimapFactory();
         public final MutableBagMultimapFactory bag = new MutableBagMultimapFactory();
+        public final MutableSortedBagMultimapFactory sortedBag = new MutableSortedBagMultimapFactory();
 
```

Comment:
This change is great, but isn't described by the commit message. I'd recommend splitting the commit into two, and covering both in the release notes draft.